integration_tests_2/README.md

```diff
@@ -0,0 +1,3 @@
+## Test dbt Project
+
+The models within this folder are used for testing use cases that can't be checked as part of the other `integration_tests` project. No tests are defined as the main purpose is to ensure that all models run properly in specific situations.
```

Comment:
Can we define what the edge cases are that this is capturing?

- no metrics
- no exposures
- no exceptions seed